cEP-0005.md

```diff
@@ -124,6 +124,15 @@ class RedundancyBear(LocalBear):
             yield Result.from_values(aspect=aspect, ...)
 ```
 
+## Aspects
+An aspect should just talk about a property of something, not qualifying it. Don't name it TooLong but Length.
+Then give it settings, e.g. Metadata.CommitMessage.Shortlog.Length can have a setting max_shortlog_length.
+
+## Settings
```

Comment:
I think its better we have definition of tastes instead of settings. Moreover which issue is this related to? Have a proper commit message.